deployment/appInsights.md

```diff
@@ -0,0 +1,20 @@
+## What is application insights?
+Application insights allows you to get actionable insights through application performance management and instant analytics. It helps you seamlessly integrate with your DevOps pipeline using Visual Studio Team Services and Github
+
+Deploying Application Insights:
+In the inception.sh script, "deployAppInsights.sh" is run. In this script, we run the following command, adding application insights to your resource group:
```

Comment:
you can wrap the inception.sh and deployAppInsights.sh in ``.

ie
`inception.sh`